smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/autocompound.rs

```diff
@@ -366,60 +402,61 @@ fn test_autocompound_with_rewards_swap_non_vault_funds() {
     )
     .unwrap();
 
-    // Assert balances after autocompound of funds
-    let (expected_refund_base, expected_refund_quote) =
-        calculate_expected_refunds(49500000000, 49500000000, deposit_ratio);
-    // Base about rewards swapped to token base
-    let balances_after_autocompound_base =
+    // Get contract balances after the autocompound
```

Comment:
unnecessary comment. If we don't read the balance, why do we call get_balance_amount?